machine_metadata.yaml

```diff
@@ -15,7 +15,7 @@ website: https://charmhub.io/grafana-agent
 source: https://github.com/canonical/grafana-agent-k8s-operator
 issues: https://github.com/canonical/grafana-agent-k8s-operator/issues
 
-series: []
+series: [jammy]
```

Comment:
This was needed to "fix" this:
```
ERROR cannot add relation "agent:metrics-endpoint kafka:metrics-endpoint": principal and subordinate applications' series must match
```

Reply:
We probably shouldn't be relating there anyway, but this is a whole new world for us. It [doesn't look that particular](https://github.com/juju/juju/blob/develop/state/state.go#L2232). I think we may actually be able to expand this to "any series which this charm will work as a subordinate of."

Considering that we aren't actually messing with dpkg/packages at all, or compiled code (which would be sort-of fine in a snap anyway), I suspect it's reasonable to have `[focal, jammy]`, at least.